nbs/examples/MLFlow_and_neuralforecast.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #59.    # Registering model
</pre><p>I don't think we need this part for this tutorial</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/892/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I tested this suggestion. However, by removing the #Registering the model code section, it is not possible to get MLFlow UI to actually see the model and record the experiments, which also allow you to visualize metrics, compare model runs, etc. Thus, this should stay to retain the full benefit gained by training, tracking and saving models with MLFLow.

Reply:
I see. Maybe we can just remove the infer_signature part. That was for the predictions, right?

Reply:
<p>Good point, I removed this line of code as suggested. I also removed "from mlflow.models import infer_signature" from the top on this cell because this import is not longer needed. Refer to commit <a href="https://github.com/Nixtla/neuralforecast/pull/892/commits/eb0ab96e1323fc1bdd12fa13ce0fd891ba54d889" target="_blank">https://github.com/Nixtla/neuralforecast/pull/892/commits/eb0ab96e1323fc1bdd12fa13ce0fd891ba54d889</a>  </p>

Reply:
I just tried this and I'm able to see the experiment, just not the model. But I think that's fine since the model isn't that helpful by itself and we're already saving the NeuralForecast object.